### ISSUE TITLE
Request action box cannot be closed with keyboard

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-app-rdm#1647]
Also, keyboard navigation doesn't work. Focus is not put in input field, and you cannot tab to the buttons

![Screenshot 2022-05-19 at 23 38 36](https://user-images.githubusercontent.com/1698163/169408954-629334b2-15c4-464e-a9c1-be8c144edd47.png)

**Comment by lnielsen** [comment]
Also, the cancel action looks strange with 2 times cancel:

![Screenshot 2022-05-19 at 23 40 21](https://user-images.githubusercontent.com/1698163/169409273-652550bd-d1f6-4a95-b8a9-292f7f2a508c.png)

**Comment by jennur** [comment]
I did not do anything about the "Cancel request"-button, it currently looks like this. LMK if it should still be updated as well! 

<img width="694" alt="Screenshot 2022-05-20 at 17 38 35" src="https://user-images.githubusercontent.com/21052053/169563205-8ed3f02e-fdfa-4172-92ff-3afcf514affd.png">
